### PR TITLE
Add components for sending email on collection manager changes

### PR DESCRIPTION
**Pull request by aaron-collier**
## Why was this change made?

Adding mailing for manager changes.

TODO: tests

### CODE REVIEW

File: app/models/collection.rb [hunk 1]
```diff
@@ -62,15 +62,25 @@ def purl
                            .invitation_to_deposit_email.deliver_later
         end
 
-        change_set.removed_depositors.each do |depositor|
```
Comment (aaron-collier) [inline app/models/collection.rb:65]: this just moved down for alpha ordering

File: app/models/collection.rb [hunk 2]
```diff
@@ -62,15 +62,25 @@ def purl
...
-          CollectionsMailer.with(collection: collection, user: depositor)
-                           .deposit_access_removed_email.deliver_later
+        change_set.added_managers.each do |manager|
+          CollectionsMailer.with(collection: collection, user: manager)
```
Comment (mjgiarlo) [inline app/models/collection.rb:66]: Looks like lines 66 and 81 are uncovered which may be why CC is angry.

**Comment by jcoyne** [comment]
Can you please update the description to add the issue this is relevant to?

**Comment by aaron-collier** [comment]
Closing in favor of #875

**Pull request closed without merging**